--- raw-thumbnailer-3.0.0/configure.ac.orig	2011-12-17 07:43:11.000000000 +0100
+++ raw-thumbnailer-3.0.0/configure.ac	2016-12-11 20:12:44.661416746 +0100
@@ -22,7 +22,9 @@
 AC_SUBST(GNOME_CFLAGS)
 AC_SUBST(GNOME_LIBS)
 
-PKG_CHECK_MODULES(OPENRAW, libopenraw-gnome-1.0 >= 0.0.9)
+PKG_CHECK_MODULES(OPENRAW, libopenraw-gnome-0.2 >= 0.0.9, [],
+  [PKG_CHECK_MODULES(OPENRAW, libopenraw-gnome-0.1 >= 0.0.9, [],
+     [PKG_CHECK_MODULES(OPENRAW, libopenraw-gnome-1.0 >= 0.0.9)])])
 AC_SUBST(OPENRAW_CFLAGS)
 AC_SUBST(OPENRAW_LIBS)
 
